parm/config/gfs/config.resources

```diff
@@ -533,78 +624,59 @@ case ${step} in
     ;;
 
   "anal")
-    export wtime_anal="01:20:00"
+    export wtime_anal_gdas="01:20:00"
     export wtime_anal_gfs="01:00:00"
-    export npe_anal=780
-    export nth_anal=5
-    export npe_anal_gfs=825
-    export nth_anal_gfs=5
-    if [[ "${machine}" == "WCOSS2" ]]; then
-      export nth_anal=8
-      export nth_anal_gfs=8
-    fi
     case ${CASE} in
+      "C768")
+        export npe_anal_gdas=780
+        export npe_anal_gfs=825
+        export nth_anal=5
+        ;;
       "C384")
-        export npe_anal=160
+        export npe_anal_gdas=160
         export npe_anal_gfs=160
         export nth_anal=10
-        export nth_anal_gfs=10
-        if [[ ${machine} = "HERA" ]]; then
-          export npe_anal=270
-          export npe_anal_gfs=270
-          export nth_anal=8
-          export nth_anal_gfs=8
-        fi	  
-        if [[ ${machine} = "S4" ]]; then
-          #On the S4-s4 partition, this is accomplished by increasing the task
-          #count to a multiple of 32
-          if [[ ${PARTITION_BATCH} = "s4" ]]; then
-            export npe_anal=416
-            export npe_anal_gfs=416
-          fi
-          #S4 is small, so run this task with just 1 thread
-          export nth_anal=1
-          export nth_anal_gfs=1
-          export wtime_anal="02:00:00"
-        fi
         ;;
       "C192" | "C96" | "C48")
-        export npe_anal=84
+        export npe_anal_gdas=84
         export npe_anal_gfs=84
-        if [[ ${machine} == "S4" ]]; then
-          export nth_anal=4
-          export nth_anal_gfs=4
-          #Adjust job count for S4
-          if [[ ${PARTITION_BATCH} == "s4" ]]; then
-            export npe_anal=88
-            export npe_anal_gfs=88
-          elif [[ ${PARTITION_BATCH} == "ivy" ]]; then
-            export npe_anal=90
-            export npe_anal_gfs=90
-          fi
-        fi
+        export nth_anal=5
         ;;
       *)
-        echo "FATAL ERROR: Resources not defined for job ${job} at resolution ${CASE}"
+        echo "FATAL ERROR: Resources not defined for job ${step} at resolution ${CASE}"
         exit 4
         ;;
     esac
     export npe_node_anal=$(( npe_node_max / nth_anal ))
     export nth_cycle=${nth_anal}
     export npe_node_cycle=$(( npe_node_max / nth_cycle ))
     export is_exclusive=True
+
+    var_npe="npe_anal_${RUN}"
+    if [[ -n "${!var_npe+0}" ]]; then
+      declare -x "npe_anal"="${!var_npe}"
+    fi
     ;;
 
   "analcalc")
     export wtime_analcalc="00:10:00"
+    [[ ${CASE} == "C384" || ${CASE} == "C768" ]] && export wtime_analcalc="00:15:00"
```

Comment:
I'm not worried about shaving five minutes on the job request when running low-resolution.
```suggestion
    export wtime_analcalc="00:15:00"
```